### PR TITLE
feat(server): switch to another node host when current is down.

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

**Comment by classicalliu** [comment]
@shaojunda CI failed

**Comment by shaojunda** [comment]
fixed please review. @classicalliu

File: server/config/settings/development.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+hosts:
+  - "http://localhost:8114"
```
Comment (classicalliu) [inline server/config/settings/development.yml]: If localhost is default to all environments, it's better to set in `config/settings.yml`

Comment (shaojunda) [reply server/config/settings/development.yml]: Thank you, done.

**Pull request merged**